xla/service/gpu/runtime/gpublas_lt_matmul.cc

```diff
@@ -105,22 +105,30 @@ absl::Status DoMatmul(
 
   // Get the matmul plan for this instance of matmul: by default we allocate it 
   // for 8 devices (which should be enough for most cases).
-  TF_ASSIGN_OR_RETURN(auto planVec, matmul_plan.GetOrCreate([] {
-    return MatmulPlanVec(8); 
+  TF_ASSIGN_OR_RETURN(auto plan_vec, matmul_plan.GetOrCreate([] {
+    return absl::StatusOr< MatmulPlanVec >(8);
```

Comment:
It can be 16 - there are 16 GPU machines.

Reply:
I have changed it to 16: actually it's just a matter of how much memory initially allocated to MatmulPlanVec. I thought that for the most systems 8 GPUs is enough but, of course, the code can handle cluster with any number of GPUs

Reply:
Yes, I understand, thanks.